### DISCUSSION TITLE
PowerModels Package Naming

### DISCUSSION THREAD

**Discussion by friederikemeier** [e2nIEE/pandapower#1069] [General]
Hey!

We are planning to put all the julia stuff into one Julia package.
And this package needs a name. The current suggestion is
PpPmConverter, but I am not 100% percent satisfied with that.. Any other suggestions @MMajidi137 ??

**Comment by MMajidi137** [comment]
what about PmOpf.jl ? or we can mix the names, for example. PandaModels.jl?

**Reply by friederikemeier** [reply]
I think Opf in the name is misleading, as it is not exclusively for OPF calculations. PpPmInterface.jl is my suggestion, but also not 100% satisfied with that now.. Anyway, guten Rutsch!
